### PR TITLE
docs: document new usage patterns

### LINKED ISSUE #24: Document public editor API
__What should we do?__

* [ ] Document public API

---

Depends on https://github.com/bpmn-io/form-js/issues/19

### LINKED ISSUE #106: Update documentation to reflect new usage pattern
__What should we do?__

* [ ] Update toolkit documentation (readme) to reflect new usage pattern
* [ ] Update [examples](https://github.com/bpmn-io/form-js-examples) to reflect new usage patterns

__Why should we do it?__

Phase out the `create*` things.

### PR DESCRIPTION
**Pull request by nikku**
Closes #106
Closes #24

### CODE REVIEW

**Review by philippfromme — COMMENTED** [review]
I've fixed some minor things and documented the editor API, too. Can you have a look?

File: packages/form-js-editor/README.md [hunk 1]
```diff
@@ -10,6 +10,112 @@ npm install @bpmn-io/form-js-editor
...
+## API
+
+### `FormEditor`
+
+Create a new form editor with options `{ container: HTMLElement }`.
+
+```javascript
+import { FormEditor } from '@bpmn-io/form-js-editor';
+
+const form = new FormEditor({
```
Comment (nikku) [inline packages/form-js-editor/README.md:67]: Should be `formEditor`.

File: packages/form-js-editor/README.md [hunk 2]
```diff
@@ -10,6 +10,112 @@ npm install @bpmn-io/form-js-editor
...
+}
+```
+
+
+### `Form#saveSchema() => Schema`
+
+Export the form schema.
+
+```javascript
+const schema = formeditor.saveSchema(schema);
```
Comment (nikku) [inline packages/form-js-editor/README.md:91]: Should be `formEditor`?

File: packages/form-js-editor/README.md [hunk 3]
```diff
@@ -10,6 +10,112 @@ npm install @bpmn-io/form-js-editor
...
+```
+
+
+### `Form#importSchema(schema: Schema) => Promise<Result, Error>`
+
+Display and edit a form represented via a form schema.
+
+```javascript
+try {
+  await formeditor.importSchema(schema);
```
Comment (nikku) [inline packages/form-js-editor/README.md:79]: Should be `formEditor`?

**Review by nikku — COMMENTED** [review]
I've found a few more issues.

File: packages/form-js-editor/README.md (hunk 1 above)
Comment (philippfromme) [reply packages/form-js-editor/README.md:67]: Argh.

Comment (philippfromme) [reply packages/form-js-editor/README.md:67]: Will fix. 🛠️

**Comment by philippfromme** [comment]
Should be good now. 👍🏻

**Review by philippfromme — APPROVED** [review]
💎

**Pull request merged**